### PR TITLE
Added task management routes

### PR DESCRIPTION
**Pull request by jesse-mw**
Added a GET route for task management index page.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -4,4 +4,8 @@ const port = process.env.PORT || 3000
 
 app.get('/', (req, res) => res.send('Hello World!'))
 
+app.get("/api/secure/back-end/routes/management/tasks/", (request, response) => {
```
Comment (genagain) [inline app.js]: Could you please explain your rationale for naming this route? It doesn't seem to follow a convention.

Comment (jesse-mw) [reply app.js]: Hey Gen! You're totally right... I've been working on one of my private projects all night that uses some realllllllly long routing like this, my bad! I'll push up a change to meet conventions.

Comment (genagain) [reply app.js]: Excellent! Thank you @jessewarren-aa

**Review by genagain — APPROVED** [review]
LGTM! 🚀

**Pull request merged**